core/src/test/java/org/bitcoinj/core/TransactionTest.java

```diff
@@ -179,11 +182,26 @@ public void testCLTVPaymentChannelTransactionSpending() {
         tx.getInput(0).setSequenceNumber(0);
         tx.setLockTime(time.subtract(BigInteger.ONE).longValue());
         TransactionSignature fromSig =
-                tx.calculateSignature(0, from, outputScript, Transaction.SigHash.SINGLE, false);
+                tx.calculateSignature(
```

Comment:
Looks like you re-formatted quite a bit of existing tests. Generally, I would like to avoid this, as it bloats the final "segwit" commit. For reformatting existing code, I would prefer a separate PR for that (against master).